compiler/src/dmd/expression.d

```diff
@@ -1553,15 +1553,15 @@ extern (C++) abstract class Expression : ASTNode
      * Check that the postblit is callable if t is an array of structs.
      * Returns true if error happens.
      */
-    extern (D) final bool checkPostblit(Scope* sc, Type t)
+    extern (D) final bool checkPostblit(Scope* sc, Type t, const ref Loc loc)
```

Comment:
I added this parameter because this function limits the knowledge of the compiler to the called type itself and the expression information is lost by the time the TypeInfo is requested

Reply:
Sorry but I don't understand the reasoning. `checkPostblit` is a member of expression which stores the location which you are passing here through parameters. What I mean is that if you delete this parameter and still pass `loc` at line 1564 you will achieve the same behavior.